react-hyf/server/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js",
+    "serve": "nodemon server.js",
+    "dev": "concurrently \"npm run serve\" \"cd ../react-hyf-client && npm start\""
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "concurrently": "^4.1.1",
+    "express": "^4.17.1",
+    "path": "^0.12.7"
```

Comment:
Why would you install path as an npm package if you have it as a core? either way you haven't used it 🙃